README.md

```diff
@@ -1,2 +1,31 @@
-# enumerable
-In this project we applied our learning skills about how to use a module inside a class.
+# Getting Started
+
```

Comment:
- [ ] It would be better if we have a descriptive title and description for our project. Also remember to add instructions about how to run the app locally in the README file to make the README professional